workflow-manager/main.go

```diff
@@ -270,14 +284,17 @@ func main() {
 			aggregationPeriod:       aggregationPeriodParsed,
 			gracePeriod:             gracePeriodParsed,
 		})
+		if err != nil {
+			log.Err(err).Str("aggregation ID", aggregationID).Msgf("Failed to schedule aggregation tasks: %s", err)
+		}
```

Comment:
omfg was I not checking this error?! Wow, I'm really glad you caught this! Besides logging the error, this branch should return, to avoid incorrectly setting `workflowManagerLastSuccess`.